Geometry/HGCalCommonData/src/HGCalDDDConstants.cc

```diff
@@ -1194,10 +1223,10 @@ std::pair<int, int> HGCalDDDConstants::waferTypeRotation(int layer, int waferU,
       }
     }
   }
-#ifdef EDM_ML_DEBUG
-  k edm::LogVerbatim("HGCalGeom") << "waferTypeRotation: Layer " << layer << " Wafer " << waferU << ":" << waferV
-                                  << " Type " << type << " Rotation " << rotn;
-#endif
+  if (debug)
```

Comment:
what's the reason to put these messages behind a run-time switch instead of a compile-time flag?

Reply:
I wanted to debug this without recompiling a part